content/operate/rc/databases/rdi/provision.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: Provision data pipeline
+alwaysopen: false
+categories:
+- docs
+- operate
+- rc
+description: Provision and troubleshoot your data pipeline infrastructure.
+hideListLinks: true
+weight: 2
+---
+
+After you have [prepared your source database]({{<relref "/operate/rc/databases/rdi/setup">}}) and connection information, you can set up your new pipeline.
+
+1. In the [Redis Cloud console](https://cloud.redis.io/), go to your target database and select the **Data Pipeline** tab.
+1. Select **Start pipeline setup**. 
+1. Enter a **Pipeline name**. This pipeline name will be the prefix to all keys generated by this pipeline in the target database.
+1. Enter the **Deployment CIDR** for your pipeline, or use the one generated for you. This CIDR should not conflict with your apps or other databases.
+1. In the **Connectivity** section, enter the **PrivateLink service name** of the [PrivateLink connected to your source database]({{< relref "/operate/rc/databases/rdi/setup#set-up-connectivity" >}}).
+1. Enter your database details. This depends on your database type, and includes:
+    - **Port**: The database's port
+    - **Database**: Your database's name, or the root database *(PostgresSQL, Oracle only)*; or a comma-separated list of one or more databases you want to connect to *(SQL Server only)*
+    - **Database Server ID**: Unique ID for the replication client. Leave as default if you don't use replication *(mySQL and mariaDB only)*
+    - **PDB**: Name of the Oracle pluggable database *(Oracle only)*
+1. Enter the ARN of your [database credentials secret]({{< relref "/operate/rc/databases/rdi/setup#share-source-database-credentials" >}}) in the **Source database secrets ARN** field.
+1. Select **Start pipeline setup**.
+
+At this point, Redis Cloud will provision the pipeline infrastructure that will allow you to define your data pipeline. 
+
+Pipelines are provisioned in the background. You aren't allowed to make changes to your data pipeline or to your database during provisioning. This process will take a long time, so you can close the window and come back later.
+
+See [Pipeline provisioning errors](#errors) to view a list of errors that can occur at this point.
+
+When your pipeline is provisioned, select **Complete setup**. You will then [define your data pipeline]({{<relref "/operate/rc/databases/rdi/define">}}).
+
+## Pipeline provisioning errors {#errors}
+
+Add errors here.
```

Comment:
I'm guessing you haven't been provided with the list of errors yet :-)

Reply:
Basically, yes. I need to go through the mockups again.